.travis/apisix_cli_test.sh

```diff
@@ -408,3 +408,36 @@ if ! grep "lua_shared_dict my_dict 1m;" conf/nginx.conf > /dev/null; then
 fi
 
 echo "passed: found 'my_dict' in nginx.conf"
+
+# check disable cpu affinity
+git checkout conf/config.yaml
+
+echo '
+nginx_config:
+  enable_cpu_affinity: false
+' > conf/config.yaml
+
+make init
+
+count=`grep -c "worker_cpu_affinity" conf/nginx.conf  || true`
+if [ $count -ne 0 ]; then
+    echo "failed: nginx.conf file disable cpu affinity"
```

Comment:
Should be `enable cpu ...`?

Reply:
This case is used to disable cpu affinity.

Reply:
@ShiningRush 
My word is not clear. Actually I mean the error message should show why the test is failed, for example, "failed: nginx.conf file should disable cpu affinity but it doesn't".

Reply:
Got it.